x11-wm/metacity/metacity-3.46.0.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit gnome2
+
+DESCRIPTION="GNOME Flashback window manager"
+HOMEPAGE="https://gitlab.gnome.org/GNOME/metacity/"
+
+LICENSE="GPL-2+"
+SLOT="0"
+IUSE="+libcanberra vulkan xinerama"
+KEYWORDS="~amd64 ~riscv"
+
+# TODO: libgtop could be optional, but no knob
+RDEPEND="
+	>=dev-libs/glib-2.67.3:2
+	>=x11-libs/gtk+-3.24.6:3[X]
+	>=x11-libs/pango-1.2.0
+	>=x11-libs/libXcomposite-0.3
+	>=x11-libs/libXres-1.2
```

Comment:
You added this dependency on libXres here, but there's an existing one a few lines below that should just be updated instead.